types/ember/index.d.ts

```diff
@@ -379,8 +379,20 @@ export namespace Ember {
         const htmlSafe: typeof EmberTemplateNs.htmlSafe;
         const isHTMLSafe: typeof EmberTemplateNs.isHTMLSafe;
     }
-    const computed: typeof EmberObjectNs.computed;
-    const run: typeof EmberRunloopNs.run;
+    const computed: typeof EmberObjectNs.computed & typeof EmberObjectComputedNs;
+
+    // Shenanigans to make `run` both callable and a namespace safely, while not
+    // making the `run.bind` call resolve to `Function.prototype.bind`. (Yes,
+    // insert :upside-down-smiley: here.)
+    // 1. Get the type side of the namespace.
+    type EmberRunloop = typeof EmberRunloopNs;
+    // 2. Use it to get an interface representing the callable side of `run`.
+    type RunFn = Pick<EmberRunloop, 'run'>['run'];
```

Comment:
I think you can skip the `Pick` here?

```ts
type RunFn = EmberRunloop['run'];
```

Reply:
Yeah, there was some reason we(/I?) did this that had to do with how type resolution was working at some point (the `Pick` *was* actually making a difference?), but I reran all the relevant tests and it's fine now.